apps/sample-app/src/components/PeopleAvatars.tsx

```diff
@@ -32,17 +32,11 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
 
   for (let i = 0; i < messages.length; i++) {
     const message = messages[i];
-    if (
-      message['attendees'].length > 15 ||
-      message['subject']?.includes('OOF') ||
-      message['subject']?.includes('Canceled')
-    ) {
+    // 5 is an arbitrary number to show only relevant meetings
+    if (message['attendees'].length > 5 || message['isCancelled'] || message['showAs'] === 'free') {
```

Comment:
>message['isCancelled'] || message['showAs'] === 'free' [](http://example.com/codeflow?start=43&length=54)

Since it seems like you do this two different places now (here and in Meetings.tsx) you should consider making a utility function like `ShouldShowMessage` or `IsMessageVisible` or something and then extract these two checks into that helper method and call it from the two places.

Reply:
I like this idea, will be a lot cleaner